src/components/carousel/carousel.jsx

```diff
@@ -0,0 +1,153 @@
+              </p>
+              <p className="primary-col white-trans-bg fw-b f-xxl end">Men Store
+              </p>
+            </div>
+            <div className="card-content inline fl-center">
+              <FontAwesomeIcon icon={faAnglesLeft} className="card-icon round left f-l" fixedWidth="fixedWidth" onClick={() => slideBack()}/>
+              <div className="card-content white-trans-bg stacked">
+                <p>{slide.caption}</p>
+              </div>
+              <FontAwesomeIcon icon={faAnglesRight} className="card-icon round right f-l" fixedWidth="fixedWidth" onClick={() => slideForward()}/>
+            </div>
+          </div>
+        </article>
+      </section>);
+
+    case 2:
+      return (
+        <section className="grid-two-layout-container">
+        <div className="container fl-end">
+          <article className="grid-two-layout-left">
+            <div className="card w-m">
+              <img src={slide.imgSrcLeft} alt="" className="background slider-img"></img>
+              <div className="card-content stacked">
+                <h3 className="fw-li">
+                  <a href="templates/product.html">Women</a>
+                </h3>
+                <FontAwesomeIcon icon={faAnglesLeft} className="card-icon round left f-l" fixedWidth="fixedWidth" onClick={() => slideBack()}/>
+                <p>1920</p>
+              </div>
+            </div>
+          </article>
+        </div>
+        <div className="container fl-start">
+          <article className="grid-two-layout-right">
+            <div className="card w-m">
+              <img src={slide.imgSrcRight} alt="" className="background"></img>
+              <div className="card-content stacked end">
+                <h3 className="fw-li">
+                  <a href="templates/product.html">Men</a>
+                </h3>
+                <FontAwesomeIcon icon={faAnglesRight} className="card-icon round left f-l" fixedWidth="fixedWidth" onClick={() => slideForward()}/>
+                <p>1920</p>
+              </div>
+            </div>
+          </article>
+        </div>
+      </section>);
+
+    case 3:
+      return (<section className="grid-three-layout-container">
+        <div className="container fl-end">
+          <article className="grid-three-layout-left">
+            <div className="card w-m">
+              <img src={slide.imgSrcLeft} alt="" className="background"></img>
+              <div className="card-content stacked">
+                <h3 className="fw-li">
+                  <a href="templates/product.html">Men</a>
+                </h3>
+                <FontAwesomeIcon icon={faAnglesLeft} className="card-icon round left f-l" fixedWidth="fixedWidth" onClick={() => slideBack()}/>
+                <p>1920</p>
+              </div>
+            </div>
+          </article>
+        </div>
+
+        <div className="container fl-center">
+          <article className="grid-three-layout-center">
```

Comment:
Suggestion : Make More Components, it will be easier for understanding your code :)